iati_datastore/iatilib/test/test_filter.py

```diff
@@ -820,3 +820,63 @@ def test_by_participating_org_ref(self):
         })
         self.assertIn(budget_in, budgets.all())
         self.assertNotIn(budget_not, budgets.all())
+
+
+class TestParticipatingOrgWithRoleFilter(AppTestCase):
+    def test_participating_org_role_1_ref(self):
+        act_in = fac.ActivityFactory.create(
+                participating_orgs=[
+                    fac.ParticipationFactory.build(
+                        organisation__ref=u"AAA",
+                        role="1")
+                ])
+        act_out = fac.ActivityFactory.create(
+                participating_orgs=[
+                    fac.ParticipationFactory.build(
+                        organisation__ref=u"BBB",
+                        role="1")
```

Comment:
If we are testing whether the filters only apply to participating orgs with the relevant `role`, perhaps this should be
```python
organisation__ref=u"AAA",
role="2"
```

Reply:
we should test both, done